runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/internal/ReconfigureHttpIT.java

```diff
@@ -0,0 +1,149 @@
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.engine.internal;
+
+import static io.aklivity.zilla.runtime.engine.EngineConfiguration.ENGINE_DRAIN_ON_CLOSE;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.rules.RuleChain.outerRule;
+
+import java.util.concurrent.CountDownLatch;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+import io.aklivity.zilla.runtime.engine.test.EngineRule;
+import io.aklivity.zilla.runtime.engine.test.annotation.Configuration;
+import io.aklivity.zilla.runtime.engine.test.annotation.Configure;
+
+
+public class ReconfigureHttpIT
+{
+    public static final String ENGINE_CONFIG_POLL_INTERVAL_SECONDS = "zilla.engine.config.poll.interval.seconds";
+
+    private final K3poRule k3po = new K3poRule()
+        .addScriptRoot("net", "io/aklivity/zilla/specs/engine/streams/network")
+        .addScriptRoot("app", "io/aklivity/zilla/specs/engine/streams/application");
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(10, SECONDS));
+
+    private final EngineRule engine = new EngineRule()
+        .directory("target/zilla-itests")
+        .commandBufferCapacity(1024)
+        .responseBufferCapacity(1024)
+        .counterValuesBufferCapacity(8192)
+        .configure(ENGINE_DRAIN_ON_CLOSE, false)
+        .configurationRoot("io/aklivity/zilla/runtime/engine/internal")
+        .external("app0")
+        .external("app1")
+        .external("app2")
+        .clean();
+
+    @Rule
+    public final TestRule chain = outerRule(k3po).around(engine).around(timeout);
+
+    @Before
+    public void setupRegisterLatch() throws Exception
+    {
+        //Make sure that the initial configuration has finished
+        EngineTest.TestEngineExt.registerLatch.await();
+        //Register new CountDownLatch
+        EngineTest.TestEngineExt.registerLatch = new CountDownLatch(1);
+    }
+
+    @Test
+    @Configure(name = ENGINE_CONFIG_POLL_INTERVAL_SECONDS, value = "1")
+    @Configuration("http://localhost:8080/")
+    @Specification({
+        "${app}/reconfigure.modify.via.http/server",
+        "${net}/reconfigure.modify.via.http/client"
+    })
+    public void shouldReconfigureWhenModifiedHTTP() throws Exception
```

Comment:
Please avoid UPPERCASE words in method names, so `Http` instead of `HTTP`.

Kindly verify all other method names in this class with the same feedback.